meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx6ullevk/lmp.cfg

```diff
@@ -16,22 +19,28 @@ CONFIG_SPL_DM=y
 CONFIG_SPL_DM_SEQ_ALIAS=y
 CONFIG_SPL_OF_TRANSLATE=y
 CONFIG_SPL_MMC_SUPPORT=y
+CONFIG_SPL_MMC_TINY=y
 CONFIG_SPL_SERIAL_SUPPORT=y
 CONFIG_SPL_GPIO_SUPPORT=y
-CONFIG_SPL_POWER_SUPPORT=y
 CONFIG_SPL_SYS_MALLOC_SIMPLE=y
+CONFIG_LOGLEVEL=2
+CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x18a
+# CONFIG_SPL_BLK is not set
+# CONFIG_SPL_CRC32_SUPPORT is not set
 # CONFIG_SPL_LEGACY_IMAGE_SUPPORT is not set
 # CONFIG_SPL_EFI_PARTITION is not set
 # CONFIG_SPL_PARTITION_UUIDS is not set
 # CONFIG_SPL_DOS_PARTITION is not set
 # CONFIG_SPL_DM_GPIO is not set
-# CONFIG_SPL_DM_USB is not set
```

Comment:
Should `CONFIG_SPL_DM_USB` be removed?

Reply:
Ah good catch.  This can probably stay.  We just need the following (and they are not dependent on DM_USB in SPL):
```
CONFIG_SPL_USB_HOST_SUPPORT=y
CONFIG_SPL_USB_GADGET=y
CONFIG_SPL_USB_SDP_SUPPORT=y
```

NOTE: This might be what is contributing to push SPL size over the limit.

Reply:
@igoropaniuk  Ah.  Sorry for the confusion.  This is the normal build lmp.cfg (not mfgtools).

This setting is now redundant due to it's dependency on `CONFIG_DM_USB` which in turn is dependent on `CONFIG_USB` (taken from drivers/usb/Kconfig):

```
config DM_USB
        bool "Enable driver model for USB"
        depends on USB && DM
...
config SPL_DM_USB
	bool "Enable driver model for USB in SPL"
	depends on SPL_DM && DM_USB
	default y
```